cpp/ql/src/Telemetry/MissingIncludes.ql

```diff
@@ -0,0 +1,12 @@
+/**
+ * @name Failed to include header file
+ * @description A count of all failed includes, grouped by filename.
+ * @kind metric
+ * @tags summary telemetry
+ * @id cpp/telemetry/failed-includes
+ */
+
+import Metrics
+
+from CppMetrics::MissingIncludeCount e
+select e.getIncludeText(), e.getValue()
```

Comment:
I'm a bit concerned that this might completely blow up on real-world projects to such an extent that the SARIF will be rejected when uploaded.

Reply:
Makes sense. I've limited the results to the top 500 - not sure if that's a sensible number or not.

Reply:
Changed to 50 after discussing with Anders.